### ISSUE TITLE
Timbersaw Chakram visual bug

### ISSUE THREAD

**Issue by mnvho** [ValveSoftware/Dota2-Gameplay#17783]
### Ability name

Chakram

### Description

The Chakram on return seems to go towards a set point on the map always. However this is just a visual bug as it does deal damage to targets in between the cast point and character on return. Have not tested in game.

On return goes somewhere...
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/43485544/95aa0dde-c512-4d9f-8aaa-59f7bc4c61c1)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/43485544/c25a8b71-c73e-4995-aa13-7f699f6fde85)

But it's just a visual bug...
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/43485544/1aaf671b-16f7-4c76-9571-bcc7537b79f0)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/43485544/d4e0f689-9dd2-4fe6-ae5a-67f0bb91ee8b)

### Example Match ID (and possibly Timestamp)

N/A

**Comment by Commander-Rose** [comment]
Yep, same issue. I also tried it in game (turbo). the first chakram keeps going to a set point (looks like the centre of the map). The 2nd chakram stays in the same position as the hero. But it does cut trees and creeps and heroes in its path. but the issue is it seems to go its entire range and come back instead of staying at target point.

it does feel like a visual cue as I can see the damage happening.

**Comment by maxjdobrei** [comment]
I have the exact same visual bug, but it only applies to the offhand item "Timberthaw Ripsaw". When I equip the immortal, Rectifier, the visual bug goes away for me. Not sure exactly what's going on (maybe they fixed it in the most recent hotfix?)

**Comment by Boda805** [comment]
+1 that I am seeing this behavior with the offhand item "Timberthaw Ripsaw" as well.

**Comment by AMKhatry** [comment]
Yup, can confirm, the "Timberthaw Ripsaw" is causing it, atleast when you equip the effect for it.
